test/src/kbasesearchengine/test/events/handler/WorkspaceEventHandlerTest.java

```diff
@@ -696,4 +703,528 @@ private void failLoadWSGetWSInfoException(final Exception toThrow, final Excepti
             TestCommon.assertExceptionCorrect(got, expected);
         }
     }
+
+    @Test
+    public void updateEventAccessGroupEvent() throws Exception {
+        final CloneableWorkspaceClient clonecli = mock(CloneableWorkspaceClient.class);
+        final WorkspaceClient cloned = mock(WorkspaceClient.class);
+        final WorkspaceClient wscli = mock(WorkspaceClient.class);
+        when(clonecli.getClientClone()).thenReturn(cloned);
+        when(clonecli.getClient()).thenReturn(wscli);
+
+        StatusEvent event = StatusEvent.getBuilder(
+                "WS",
+                           Instant.ofEpochMilli(20000),
+                           StatusEventType.PUBLISH_ACCESS_GROUP)
+                           .withNullableAccessGroupID(1)
+                           .build();
+
+        when(wscli.administer(argThat(new AdminGetWSInfoAnswerMatcher(1))))
```

Comment:
I don't think this is necessary right? All the workspace accessing code will get skipped.

Reply:
I need to mock in order to simulate the kind of updates I want to execute.

Reply:
But the mock is never called in this test, unless I misunderstand the test.

Reply:
Removed mock.